.jenkins/pipelines/Azure/Windows/Jenkinsfile

```diff
@@ -10,28 +10,28 @@ try{
+                "Win2019 Ubuntu1804 clang-10 RelWithDebInfo Linux-Elf-build LVI": { tests.windowsLinuxElfBuild(params.WS2019_DCAP_CFL_LABEL, params.UBUNTU_1804_NONSGX_LABEL, 'clang-10', 'RelWithDebInfo', 'ControlFlow') },
+                "Win2019 Sim Debug Cross Compile LVI ":                           { tests.windowsCrossCompile(params.WS2019_DCAP_CFL_LABEL, 'Debug',          'ControlFlow', '1') },
+                "Win2019 Sim RelWithDebInfo Cross Compile LVI ":                  { tests.windowsCrossCompile(params.WS2019_DCAP_CFL_LABEL, 'RelWithDebInfo', 'ControlFlow', '1') },
+                "Win2019 Debug Cross Compile DCAP LVI":                           { tests.windowsCrossCompile(params.WS2019_DCAP_CFL_LABEL, 'Debug',          'ControlFlow') },
+                "Win2019 RelWithDebInfo Cross Compile DCAP LVI":                  { tests.windowsCrossCompile(params.WS2019_DCAP_CFL_LABEL, 'RelWithDebInfo', 'ControlFlow') },
+                "Win2019ICX RelWithDebInfo Cross Compile DCAP LVI":               { tests.windowsCrossCompile(params.WS2019_DCAP_ICX_LABEL, 'RelWithDebInfo', 'ControlFlow') },
+                "Win2019 Sim Debug Cross Compile LVI snmalloc":                   { tests.windowsCrossCompile(params.WS2019_DCAP_CFL_LABEL, 'Debug',          'ControlFlow', '1', 'OFF', ['-DUSE_SNMALLOC=ON']) },
+                "Win2019 RelWithDebInfo Cross Compile DCAP LVI snmalloc":         { tests.windowsCrossCompile(params.WS2019_DCAP_CFL_LABEL, 'RelWithDebInfo', 'ControlFlow', '0', 'OFF', ['-DUSE_SNMALLOC=ON']) },
+                "Win2019 Cross Platform":                                         { tests.windowsCrossPlatform(params.WS2019_DCAP_CFL_LABEL) }
             ]
             parallel testing_stages
         }
     } else {
         stage("PR Testing") {
             testing_stages += [
-                "Win2019 Ubuntu1804 clang-10 RelWithDebInfo Linux-Elf-build LVI":    { tests.windowsLinuxElfBuild('acc-win2019-dcap', '18.04', 'clang-10', 'RelWithDebInfo', 'ControlFlow', 'ON') },
-                "Win2019 Sim RelWithDebInfo Cross Compile LVI ":                     { tests.windowsCrossCompile('acc-win2019', 'RelWithDebInfo', 'ControlFlow', '1', 'ON') },
-                "Win2019 Debug Cross Compile DCAP LVI":                              { tests.windowsCrossCompile('acc-win2019', 'Debug',          'ControlFlow', '0', 'ON') },
-                "Win2019 RelWithDebInfo Cross Compile DCAP LVI FULL Tests":          { tests.windowsCrossCompile('acc-win2019', 'RelWithDebInfo', 'ControlFlow') },
-                "Win2019 RelWithDebInfo Cross Compile DCAP LVI FULL Tests snmalloc": { tests.windowsCrossCompile('acc-win2019', 'RelWithDebInfo', 'ControlFlow', '0', 'OFF', ['-DUSE_SNMALLOC=ON']) }
+                "Win2019 Ubuntu1804 clang-10 RelWithDebInfo Linux-Elf-build LVI":    { tests.windowsLinuxElfBuild(params.WS2019_DCAP_CFL_LABEL, params.UBUNTU_1804_NONSGX_LABEL, 'clang-10', 'RelWithDebInfo', 'ControlFlow', 'ON') },
```

Comment:
why do you have a CFL label?

Reply:
This is so we can distinguish between ICX and CFL agents while also parameterizing the agent labels so we can run these on E2E agents as well (since for E2E agents we use the same labels as their normal counterpart, but with an "-e2e" suffix).